apis/v1alpha1/httproute_types.go

```diff
@@ -203,6 +215,17 @@ type HTTPRouteRule struct {
 // * "RegularExpression"
 // * "ImplementationSpecific"
 //
+// For Prefix and Exact paths:
+// Every Path p_i must be syntactically valid:
+// 	- Must begin with the '/' character
+// 	- Must not contain consecutive '/' characters (e.g. /foo///, //).
+// 	- For prefix paths, a trailing '/' character in the Path is ignored,
+//      e.g. /abc and /abc/ specify the same match.
+//
+// If there is more than one potential match:
+//  - Exact match is preferred to a Prefix match.
+//  - For multiple prefix matches, the longest Path p_i will be the matching path.
```

Comment:
I think this is already covered above. It would probably be best to remove this so we don't risk the two sections becoming out of sync.